cobalt/debug/remote/debug_web_server.cc

```diff
@@ -118,17 +119,19 @@ DebugWebServer::DebugWebServer(
 }
 
 DebugWebServer::~DebugWebServer() {
+  LOG(INFO) << "DebugWebServer::~DebugWebServer()";
```

Comment:
Do you want to keep these all as regular logs?

Reply:
No, this should be removed. I'll update the CL once the parent merges.